setup.py

```diff
@@ -25,6 +27,17 @@
 if py_version >= (3, 5):
     packages.append('websockets/py35')
 
+ext_modules = []
+if (platform.python_implementation() == 'CPython' and
+        os.environ.get('WEBSOCKETS_CEXT_BUILD') != '0'):
```

Comment:
I saw that and I merged part of that into this PR. For tests however, I didn't excluded anything from the coverage. I just run them twice deleting the extension for one run which shall get 100% coverage. That being said, I can still report 100% coverage with tests passing without the extension module being loaded (because gcovr doesn't have a `--fail-under`option c.f. gcovr/gcovr#173). I'll try to rework tests as you have.